index.html

```diff
@@ -0,0 +1,229 @@
+            <li>javascript</li>
+          </ul>
+          <div id="btn-curser">
+            <button type="button" class="btn" id="button">See Projects</button>
+            <div id="curser"></div>
+          </div>
+        </div>
+      </section>
+
+      <section class="card">
+        <div class="card-content">
+          <h2>Uber Navigation</h2>
+          <div class="list-1">
+            <p>Uber</p>
+            <ul class="field">
+              <li class="backend">Back End Dev</li>
+              <li class="year">2018</li>
+            </ul>
+          </div>
+
+          <p>
+            A smart assistant to make driving more safe, efficient, and fun by
+            unlocking your most expensive computer: your car.
+          </p>
+          <ul class="languages">
+            <li>html</li>
+            <li>Ruby on rails</li>
+            <li>css</li>
+            <li>javascript</li>
+          </ul>
+          <button type="button" class="btn">See Projects</button>
+        </div>
+        <img src="images/Snapshoot-Portfolio3.png" alt="" />
+      </section>
+      
+      <section id="ab-lang">
+        <div id="about1">
+            <h3>
+                About <br />
+                Myself
+              </h3>
+              <p>
+                Hello I’m a software developer! I can help you build a product ,
+                feature or website Look through some of my work and experience! If you
+                like what you see and have a project you need coded, don’t hestiate to
+                contact me.
+              </p>
+      
+              <div id="about-social">
+                <p>Let's connect</p>
+                <ul class="social-icons">
+                  <li><i class="fa-brands fa-twitter"></i></li>
+                  <li><i class="fa-brands fa-linkedin"></i></li>
+                  <li><i class="fa-brands fa-medium"></i></li>
+                  <li><i class="fa-brands fa-github"></i></li>
+                  <li><i class="fa-brands fa-angellist"></i></li>
+                </ul>
+              </div>
+              
+              <button type="button" id="get-resume">Get my resume</button>      
+            </div>  
+  
+            <div class="lang">
+          <ul class="showcase">
+            <li class="showcase-list">
+              <p>Languages</p>
+              <i class="fas fa-caret-down accordion"></i>
+            </li>
+            <li>
+              <ul id="languages">
+                <li class="language">
+                  <img src="./images/javascript.png" alt="JavaScript" />
+                  <p>JavaScript</p>
+                </li>
+                <li class="language">
+                  <img src="./images/html5.png" alt="JavaScript" />
+                  <p>HTML</p>
+                </li>
+                <li class="language">
+                  <img src="./images/css3.png" alt="JavaScript" />
+                  <p>CSS</p>
+                </li>
+              </ul>
```

Comment:
- [ ] Please notice the boxes containing each of the language icons seems to small in terms of it's width. please adjust this so it matches the Figma design.

Your Design | Reference Design
-|-
![image](https://user-images.githubusercontent.com/18730106/155595968-a36cd0dc-322a-4c0d-91c2-666bdc608b02.png)|![image](https://user-images.githubusercontent.com/18730106/155595983-7f72cfda-366d-42be-8701-4e11aa419844.png)